products/ASC.Files/Client/src/components/Badges.js

```diff
@@ -19,130 +21,107 @@ const Badges = ({
   accessToEdit,
   showNew,
   onFilesClick,
-  onClickLock,
-  onClickFavorite,
   onShowVersionHistory,
   onBadgeClick,
   setConvertDialogVisible,
 }) => {
-  const {
-    id,
-    locked,
-    fileStatus,
-    version,
-    versionGroup,
-    title,
-    fileExst,
-  } = item;
+  const { id, locked, fileStatus, version, versionGroup, fileExst } = item;
 
-  const isFavorite = fileStatus === 32;
   const isEditing = fileStatus === 1;
   const isNewWithFav = fileStatus === 34;
   const isEditingWithFav = fileStatus === 33;
   const showEditBadge = !locked || item.access === 0;
   const isPrivacy = isPrivacyFolder && isDesktopClient;
+  const showActionsEdit = isEditing || isEditingWithFav;
+
+  const iconEdit = showActionsEdit
+    ? "/static/images/file.actions.convert.edit.doc.react.svg"
+    : "/static/images/access.edit.react.svg";
+
+  const contentBadgeVersion =
+    `V.${versionGroup}` > 999 ? "V.999+" : `V.${versionGroup}`;
```

Comment:
you must use "Version" language key

Reply:
Added a new VersionBadge key and used it.

Reply:
> you must use "Version" language key

Added a new VersionBadge key and used it.

Reply:
Added a new VersionBadge key and used it.